.github/workflows/release-pr.yml

```
@@ -0,0 +1,44 @@
+name: Release PR
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - automated-release-pr
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
+jobs:
+  manage-release-pr:
+    name: Manage release PR
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+          ref: prepare-release-script-for-automation
```

To be updated before merge:

```suggestion
          ref: main
```